src/main/java/domain/Participant.java

```diff
@@ -0,0 +1,89 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public abstract class Participant {
+
+    private final String name;
+    private final List<Card> cards;
+
+    public Participant(String name) {
```

Comment:
protected 생성자를 사용해볼까요?

```suggestion
    protected Participant(String name) {
```